rma_sale_lot/models/rma.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Rma(models.Model):
+
+    _inherit = "rma"
+
+    def _create_receipt(self):
+        res = super()._create_receipt()
```

Comment:
@sbejaoui Do you know why tests seem to not pass here ? See coverage

Reply:
_create_receipt is defined in #413 , is not merged yet.